Food.html

```diff
@@ -0,0 +1,76 @@
+</li><li class="u-nav-item"><a class="u-border-active-palette-1-base u-border-hover-palette-1-base u-button-style u-nav-link u-text-active-palette-1-base u-text-grey-90 u-text-hover-palette-2-base" href="Jobs.html" style="padding: 10px;">Jobs</a>
+</li><li class="u-nav-item"><a class="u-border-active-palette-1-base u-border-hover-palette-1-base u-button-style u-nav-link u-text-active-palette-1-base u-text-grey-90 u-text-hover-palette-2-base" href="Events.html" style="padding: 10px;">Events</a>
+</li><li class="u-nav-item"><a class="u-border-active-palette-1-base u-border-hover-palette-1-base u-button-style u-nav-link u-text-active-palette-1-base u-text-grey-90 u-text-hover-palette-2-base" href="Food.html" style="padding: 10px;">Food</a>
+</li><li class="u-nav-item"><a class="u-border-active-palette-1-base u-border-hover-palette-1-base u-button-style u-nav-link u-text-active-palette-1-base u-text-grey-90 u-text-hover-palette-2-base" href="Services.html" style="padding: 10px;">Services</a>
+</li><li class="u-nav-item"><a class="u-border-active-palette-1-base u-border-hover-palette-1-base u-button-style u-nav-link u-text-active-palette-1-base u-text-grey-90 u-text-hover-palette-2-base" href="Attractions.html" style="padding: 10px;">Attractions</a>
+</li></ul>
+          </div>
+          <div class="u-custom-menu u-nav-container-collapse">
+            <div class="u-black u-container-style u-inner-container-layout u-opacity u-opacity-95 u-sidenav">
+              <div class="u-sidenav-overflow">
+                <div class="u-menu-close"></div>
+                <ul class="u-align-center u-nav u-popupmenu-items u-unstyled u-nav-2"><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Home.html" style="padding: 10px;">Home</a>
+</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Jobs.html" style="padding: 10px;">Jobs</a>
+</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Events.html" style="padding: 10px;">Events</a>
+</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Food.html" style="padding: 10px;">Food</a>
+</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Services.html" style="padding: 10px;">Services</a>
+</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Attractions.html" style="padding: 10px;">Attractions</a>
+</li></ul>
+              </div>
+            </div>
+            <div class="u-black u-menu-overlay u-opacity u-opacity-70"></div>
+          </div>
+        </nav>
+      </div></header> 
+    
+    
+    <footer class="u-align-center u-clearfix u-footer u-grey-80 u-footer" id="sec-aba3"><div class="u-clearfix u-sheet u-valign-middle u-sheet-1">
+        <p class="u-small-text u-text u-text-variant u-text-1">event</p>
+      </div></footer>
+  </body>
+</html>
```

Comment:
Overall code looks good and has consistent style, but could use some comments to make code easier to read

Reply:
needs more comments